core-bundle/src/Resources/contao/languages/en/tl_maintenance.xlf

```diff
@@ -116,6 +92,30 @@
       <trans-unit id="tl_maintenance_jobs.symlinks.1">
         <source>Recreates the symlinks in the &lt;code&gt;web/&lt;/code&gt; folder (document root).</source>
       </trans-unit>
+      <trans-unit id="tl_maintenance.crawl.subscribersLabel.0">
```

Comment:
Are you sure that four nesting levels actually work? Is it necessary to group them?

Reply:
It's just an array and if it would not work you would see that there are no labels in the back end, right? 😄

Reply:
It just works accidentally because we have to support some legacy keys with a dot in the key:

https://github.com/contao/contao/blob/10f7ea76a0e3a745d65b96a86f34d512ac99186f/core-bundle/src/Config/Loader/XliffFileLoader.php#L129-L132

However, keys should not have more than three nesting levels. We even have set up a unit test to ensure we are not using more. 😄 

https://github.com/contao/contao/blob/10f7ea76a0e3a745d65b96a86f34d512ac99186f/core-bundle/tests/Fixtures/vendor/contao/test-bundle/Resources/contao/languages/en/error.xlf#L5-L8

Reply:
Strange, because we're using `tl_maintenance`, not `tl_layout` here.
Anyway, fixed in 9a5a9e315260d69473b0e56ebbea21107d080641. Also provided the missing label for the broken link checker :)